packages/Poll/README.md

```diff
@@ -0,0 +1,22 @@
+# Poll
```

Comment:
How do you feel about the format of the other package README's? I was thinking we could follow that format so it's consistent.